controllers/servers/csi/tests/array_action/array_connection_manager_test.py

```diff
@@ -2,12 +2,13 @@
 
 from mock import patch
 
-import controller.array_action.array_connection_manager as array_connection_manager
-from controller.array_action.array_connection_manager import ArrayConnectionManager, NoConnectionAvailableException
-from controller.array_action.array_mediator_ds8k import DS8KArrayMediator
-from controller.array_action.array_mediator_svc import SVCArrayMediator
-from controller.array_action.array_mediator_xiv import XIVArrayMediator
-from controller.array_action.errors import FailedToFindStorageSystemType
+import controllers.array_action.array_connection_manager as array_connection_manager
```

Comment:
please move `tests` and `scripts` folders under `controllers`

Reply:
done